src/test/a11y/a11y.ts

```diff
@@ -20,9 +22,10 @@ function check (url: string): void {
   describe(`Page ${url}`, () => {
 
     it('should have no accessibility errors', (done) => {
-      ensurePageCallWillSucceed(url)
+      const urlWithParams = `${url}?jwt=ABC&ref=000MC000`
```

Comment:
this looks wrong, and this hack with jwt= won't work when the oauth stuff is turned on